website/docs/reference/resource-configs/bigquery-configs.md

```diff
@@ -902,3 +902,24 @@ As with most data platforms, there are limitations associated with materialized
 Find more information about materialized view limitations in Google's BigQuery [docs](https://cloud.google.com/bigquery/docs/materialized-views-intro#limitations).
 
 </VersionBlock>
+
+<VersionBlock firstVersion="1.9">
+
+## Python models
+
+The BigQuery adapter supports Python models with the following additional configuration parameters:
+
+| Parameter               | Type        | Required | Default   | Valid values     |
+|-------------------------|-------------|----------|-----------|------------------|
+| `enable_list_inference` | `<boolean>` | no       | `True`    | `True`, `False`  |
+| `intermediate_format`   | `<string>`  | no       | `parquet` | `parquet`, `orc` |
+
+### The `enable_list_inference` parameter
+The `enable_list_inference` parameter enables a PySpark data frame to read multiple records in the same operation.
+By default, this is set to `True` to support the default `intermediate_format` of `parquet`.
+
+### The `intermediate_format` parameter
+The `intermediate_format` parameter specifies which file format to use when writing records to a table. The default is `parquet`.
+This parameter became configurable when the default write method changed from `direct` to `indirect` to support partitioning and clustering.
```

Comment:
out of curiosity, can you tell me more about the 3rd sentence? is it necessary to have in the above paragraph? 

maybe it's just me but it has more of a "release note" vibe so was just wondering about placement/location and the context

Reply:
Yeah, I was conflicted on that too. It felt more locked in time instead of referential. Since we both reached that conclusion I'll remove it.